src/main/java/com/backbase/ct/bbfuel/configurator/BillPayConfigurator.java

```diff
@@ -32,4 +49,67 @@ public void ingestBillPayUser(String externalUserId) {
             log.info("Bill Pay user [{}] ingested", externalUserId);
         }
     }
+
+    public void ingestBillPayAccounts(String subscriberId, List<Account> accounts) {
```

Comment:
This method no longer needs to be public as it's only called from within this class in the `ingestBillPayUserAndAccounts` method.